code/modules/vore/resizing/resize_vr.dm

```diff
@@ -264,27 +276,30 @@ var/const/RESIZE_A_SMALLTINY = (RESIZE_SMALL + RESIZE_TINY) / 2
 						admin_attack_log(src, M, "Crushed [M.name] under foot for [damage * 10] damage.", "Was crushed under foot by [H.name] for [damage * 10] damage.", "Crushed [M.name] for [damage * 10] damage.")
 				return 1
 
-	if(src.a_intent == I_GRAB && src.canmove && !src.buckled)
-		if((src.get_effective_size() - tmob.get_effective_size()) >= 0.50)
-			now_pushing = 0
+	else if(src.a_intent == I_GRAB)
+		if((src.get_effective_size() / tmob.get_effective_size()) >= 1.5)
```

Comment:
Same as above math. 2.0 can't pick up a 1.5.
2.0/1.5 = 1.333

Reply:
What coefficient are you suggesting?

Reply:
Yes. 1.3 would work, as it'd allow 200 to pick up 150, and 150 to pick up 100.

Reply:
I initially used a quotient of 2 to keep it intuitive, but for the sake of making 200% to 150%-interaction possible, I suppose 1.3 is okay.